src/ImageRunModal.jsx

```diff
@@ -693,8 +703,16 @@ export class ImageRunModal extends React.Component {
                     <TextInput id='run-image-dialog-name'
                            className="image-name"
                            placeholder={_("Container name")}
+                           validated={dialogValues.validationFailed.containerName ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-110243-108e1b8a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)